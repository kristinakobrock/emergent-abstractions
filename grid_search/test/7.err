  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  8%|#######4                                                                                     | 4/50 [00:00<00:01, 28.89it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 23.33it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 23.62it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 24.27it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 23.94it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 23.58it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 25.26it/s] 50%|##############################################                                              | 25/50 [00:01<00:01, 24.31it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 25.77it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 26.65it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 27.55it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 25.80it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 25.69it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 26.74it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 27.56it/s]100%|############################################################################################| 50/50 [00:01<00:00, 26.82it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.73it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 20.83it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.20it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 21.55it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 24.24it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.72it/s]
