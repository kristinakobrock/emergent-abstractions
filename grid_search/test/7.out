# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6996974945068359, "acc": 0.5910658240318298, "length": 2.242128849029541, "mode": "train", "epoch": 1}
{"loss": 0.6638500094413757, "acc": 0.6285156607627869, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6602898240089417, "acc": 0.6272212266921997, "length": 1.2709778547286987, "mode": "train", "epoch": 2}
{"loss": 0.6605952978134155, "acc": 0.628710925579071, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.659579336643219, "acc": 0.6271488070487976, "length": 1.182979941368103, "mode": "train", "epoch": 3}
{"loss": 0.6603395938873291, "acc": 0.628710925579071, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6519531011581421, "acc": 0.6305882334709167, "length": 1.5924487113952637, "mode": "train", "epoch": 4}
{"loss": 0.6565580368041992, "acc": 0.627789318561554, "length": 1.65625, "mode": "test", "epoch": 4}
{"loss": 0.6223524808883667, "acc": 0.6744413375854492, "length": 2.3932290077209473, "mode": "train", "epoch": 5}
{"loss": 0.6718143224716187, "acc": 0.62933349609375, "length": 1.94921875, "mode": "test", "epoch": 5}
{"loss": 0.6028013825416565, "acc": 0.6928964257240295, "length": 2.4897971153259277, "mode": "train", "epoch": 6}
{"loss": 0.6451393365859985, "acc": 0.686309814453125, "length": 2.921875, "mode": "test", "epoch": 6}
{"loss": 0.5739685297012329, "acc": 0.7177593111991882, "length": 2.868140697479248, "mode": "train", "epoch": 7}
{"loss": 0.5776949524879456, "acc": 0.7253905534744263, "length": 3.09765625, "mode": "test", "epoch": 7}
{"loss": 0.5290310382843018, "acc": 0.7561002969741821, "length": 2.9462809562683105, "mode": "train", "epoch": 8}
{"loss": 0.5782139897346497, "acc": 0.7420410513877869, "length": 3.21484375, "mode": "test", "epoch": 8}
{"loss": 0.5102152228355408, "acc": 0.7681834697723389, "length": 2.9803593158721924, "mode": "train", "epoch": 9}
{"loss": 0.5566135048866272, "acc": 0.745501697063446, "length": 3.203125, "mode": "test", "epoch": 9}
{"loss": 0.48976650834083557, "acc": 0.7722909450531006, "length": 3.0659074783325195, "mode": "train", "epoch": 10}
{"loss": 0.5477240085601807, "acc": 0.74652099609375, "length": 3.12109375, "mode": "test", "epoch": 10}
{"loss": 0.47344285249710083, "acc": 0.7792999148368835, "length": 3.123410701751709, "mode": "train", "epoch": 11}
{"loss": 0.5088335275650024, "acc": 0.7614440321922302, "length": 3.30859375, "mode": "test", "epoch": 11}
{"loss": 0.44832131266593933, "acc": 0.7966141104698181, "length": 3.192103862762451, "mode": "train", "epoch": 12}
{"loss": 0.5304487347602844, "acc": 0.7657531499862671, "length": 3.3203125, "mode": "test", "epoch": 12}
{"loss": 0.43557873368263245, "acc": 0.8053695559501648, "length": 3.323345422744751, "mode": "train", "epoch": 13}
{"loss": 0.5266080498695374, "acc": 0.7669983506202698, "length": 3.23828125, "mode": "test", "epoch": 13}
{"loss": 0.4265577793121338, "acc": 0.8092842102050781, "length": 3.3399391174316406, "mode": "train", "epoch": 14}
{"loss": 0.48649200797080994, "acc": 0.772778332233429, "length": 3.484375, "mode": "test", "epoch": 14}
{"loss": 0.41343972086906433, "acc": 0.8176197409629822, "length": 3.430145025253296, "mode": "train", "epoch": 15}
{"loss": 0.4899139106273651, "acc": 0.7789307236671448, "length": 3.58984375, "mode": "test", "epoch": 15}
{"loss": 0.3940994441509247, "acc": 0.8243564367294312, "length": 3.5671873092651367, "mode": "train", "epoch": 16}
{"loss": 0.4856901466846466, "acc": 0.776507556438446, "length": 3.82421875, "mode": "test", "epoch": 16}
{"loss": 0.38784274458885193, "acc": 0.8306100368499756, "length": 3.7048778533935547, "mode": "train", "epoch": 17}
{"loss": 0.49265509843826294, "acc": 0.7957519292831421, "length": 3.9296875, "mode": "test", "epoch": 17}
{"loss": 0.38177967071533203, "acc": 0.8330817222595215, "length": 3.8155298233032227, "mode": "train", "epoch": 18}
{"loss": 0.4925655424594879, "acc": 0.7853149771690369, "length": 3.90625, "mode": "test", "epoch": 18}
{"loss": 0.36748868227005005, "acc": 0.8382385969161987, "length": 3.813295602798462, "mode": "train", "epoch": 19}
{"loss": 0.47321632504463196, "acc": 0.8002380132675171, "length": 3.94140625, "mode": "test", "epoch": 19}
{"loss": 0.36560022830963135, "acc": 0.8410911560058594, "length": 3.828327178955078, "mode": "train", "epoch": 20}
{"loss": 0.4778457581996918, "acc": 0.8035767078399658, "length": 3.94140625, "mode": "test", "epoch": 20}
{"loss": 0.3381340503692627, "acc": 0.8549630045890808, "length": 3.839487075805664, "mode": "train", "epoch": 21}
{"loss": 0.4315885603427887, "acc": 0.8153442740440369, "length": 3.98828125, "mode": "test", "epoch": 21}
{"loss": 0.3201439082622528, "acc": 0.8662667870521545, "length": 3.9253721237182617, "mode": "train", "epoch": 22}
{"loss": 0.42361435294151306, "acc": 0.8362610340118408, "length": 3.98828125, "mode": "test", "epoch": 22}
{"loss": 0.2969937026500702, "acc": 0.8821774125099182, "length": 3.9696054458618164, "mode": "train", "epoch": 23}
{"loss": 0.43009239435195923, "acc": 0.8408202528953552, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.27326229214668274, "acc": 0.8938430547714233, "length": 3.9809961318969727, "mode": "train", "epoch": 24}
{"loss": 0.38383749127388, "acc": 0.8580078482627869, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.26403987407684326, "acc": 0.8974170684814453, "length": 3.988537311553955, "mode": "train", "epoch": 25}
{"loss": 0.4051249623298645, "acc": 0.8505859375, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.24926918745040894, "acc": 0.9032675623893738, "length": 3.991834878921509, "mode": "train", "epoch": 26}
{"loss": 0.3971371054649353, "acc": 0.8499999046325684, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.23236083984375, "acc": 0.9127700924873352, "length": 3.9948246479034424, "mode": "train", "epoch": 27}
{"loss": 0.3769243061542511, "acc": 0.8650389909744263, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.21946591138839722, "acc": 0.9216556549072266, "length": 3.9972760677337646, "mode": "train", "epoch": 28}
{"loss": 0.3854907751083374, "acc": 0.8679686784744263, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.2147602140903473, "acc": 0.9219033718109131, "length": 3.997663736343384, "mode": "train", "epoch": 29}
{"loss": 0.35129231214523315, "acc": 0.87890625, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.20663118362426758, "acc": 0.9276871085166931, "length": 3.9977872371673584, "mode": "train", "epoch": 30}
{"loss": 0.3636512756347656, "acc": 0.8744140267372131, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.19478419423103333, "acc": 0.9332517385482788, "length": 3.9984986782073975, "mode": "train", "epoch": 31}
{"loss": 0.3831498324871063, "acc": 0.876953125, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.18630637228488922, "acc": 0.9361063241958618, "length": 3.998372793197632, "mode": "train", "epoch": 32}
{"loss": 0.3391113877296448, "acc": 0.8962889909744263, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.1807786226272583, "acc": 0.9394090175628662, "length": 3.999006509780884, "mode": "train", "epoch": 33}
{"loss": 0.3444724678993225, "acc": 0.8953124284744263, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.17354871332645416, "acc": 0.9416093826293945, "length": 3.9989044666290283, "mode": "train", "epoch": 34}
{"loss": 0.35763058066368103, "acc": 0.8921875357627869, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.1689186841249466, "acc": 0.944532573223114, "length": 3.9990410804748535, "mode": "train", "epoch": 35}
{"loss": 0.3538168966770172, "acc": 0.8927735090255737, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.16390828788280487, "acc": 0.9458549618721008, "length": 3.998080015182495, "mode": "train", "epoch": 36}
{"loss": 0.36173203587532043, "acc": 0.8935546278953552, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.15828445553779602, "acc": 0.9494349360466003, "length": 3.9993491172790527, "mode": "train", "epoch": 37}
{"loss": 0.3639519214630127, "acc": 0.8988280892372131, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.1473289132118225, "acc": 0.9524100422859192, "length": 3.9993221759796143, "mode": "train", "epoch": 38}
{"loss": 0.3601512908935547, "acc": 0.9027343392372131, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.14294826984405518, "acc": 0.9523631930351257, "length": 3.9995641708374023, "mode": "train", "epoch": 39}
{"loss": 0.37307536602020264, "acc": 0.8992186188697815, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.1420930027961731, "acc": 0.9537338018417358, "length": 3.999547004699707, "mode": "train", "epoch": 40}
{"loss": 0.3939365744590759, "acc": 0.8894529938697815, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.1432979553937912, "acc": 0.953319251537323, "length": 3.9996583461761475, "mode": "train", "epoch": 41}
{"loss": 0.34803491830825806, "acc": 0.9064454436302185, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.13757334649562836, "acc": 0.9552839398384094, "length": 3.9995579719543457, "mode": "train", "epoch": 42}
{"loss": 0.3670298159122467, "acc": 0.9033203125, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.13985154032707214, "acc": 0.9552181363105774, "length": 3.9994752407073975, "mode": "train", "epoch": 43}
{"loss": 0.35353487730026245, "acc": 0.9037109017372131, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.13350170850753784, "acc": 0.9548093676567078, "length": 3.9996049404144287, "mode": "train", "epoch": 44}
{"loss": 0.32966506481170654, "acc": 0.9093751311302185, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.13142316043376923, "acc": 0.956780195236206, "length": 3.999722480773926, "mode": "train", "epoch": 45}
{"loss": 0.3627997636795044, "acc": 0.9007812142372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.1330891251564026, "acc": 0.9573792815208435, "length": 3.9997918605804443, "mode": "train", "epoch": 46}
{"loss": 0.3420645594596863, "acc": 0.9085937142372131, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.12215679883956909, "acc": 0.9605365991592407, "length": 3.9997615814208984, "mode": "train", "epoch": 47}
{"loss": 0.3942342698574066, "acc": 0.8980469107627869, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.12368936836719513, "acc": 0.960417628288269, "length": 3.999763250350952, "mode": "train", "epoch": 48}
{"loss": 0.35192781686782837, "acc": 0.9058594107627869, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.11580724269151688, "acc": 0.9637567400932312, "length": 3.9997994899749756, "mode": "train", "epoch": 49}
{"loss": 0.33453866839408875, "acc": 0.9109374284744263, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.11642875522375107, "acc": 0.9632825255393982, "length": 3.999846935272217, "mode": "train", "epoch": 50}
{"loss": 0.3613337278366089, "acc": 0.9042969346046448, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.12639842927455902, "acc": 0.9604834914207458, "length": 3.9998931884765625, "mode": "train", "epoch": 51}
{"loss": 0.3635736405849457, "acc": 0.9027343392372131, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.1278034895658493, "acc": 0.9588155150413513, "length": 3.999899387359619, "mode": "train", "epoch": 52}
{"loss": 0.3622993528842926, "acc": 0.9039062857627869, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.1282295137643814, "acc": 0.9567877054214478, "length": 3.999882936477661, "mode": "train", "epoch": 53}
{"loss": 0.3518680930137634, "acc": 0.9107422232627869, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.14327490329742432, "acc": 0.9520699977874756, "length": 3.9997172355651855, "mode": "train", "epoch": 54}
{"loss": 0.37897634506225586, "acc": 0.9048828482627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.13160443305969238, "acc": 0.9554048180580139, "length": 3.9996538162231445, "mode": "train", "epoch": 55}
{"loss": 0.370124489068985, "acc": 0.9083985090255737, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.12340474128723145, "acc": 0.9586894512176514, "length": 3.9997668266296387, "mode": "train", "epoch": 56}
{"loss": 0.35730916261672974, "acc": 0.9115234017372131, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.11297892779111862, "acc": 0.9631667137145996, "length": 3.9999074935913086, "mode": "train", "epoch": 57}
{"loss": 0.3870043158531189, "acc": 0.8984375, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.11527370661497116, "acc": 0.9635845422744751, "length": 3.9999403953552246, "mode": "train", "epoch": 58}
{"loss": 0.3830866813659668, "acc": 0.9021483659744263, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.11336630582809448, "acc": 0.9641769528388977, "length": 3.999890089035034, "mode": "train", "epoch": 59}
{"loss": 0.35087230801582336, "acc": 0.9113281965255737, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.10956129431724548, "acc": 0.9642976522445679, "length": 3.999882459640503, "mode": "train", "epoch": 60}
{"loss": 0.3449268043041229, "acc": 0.9152344465255737, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7945312857627869
